tox.ini

```diff
@@ -17,7 +17,8 @@ deps = -rrequirements/test-cpu.txt
 commands =
     python -m pip install --upgrade git+https://github.com/NVIDIA-Merlin/core.git@{posargs:main}
     python -m pip install .
-    
+    python -m pip install --upgrade git+https://github.com/karlhigley/core.git@feature/merlin-dtypes
```

Comment:
I will approve but I think we want to change this back to use the main branch before integrating right?